code/modules/mob/living/living.dm

```diff
@@ -839,12 +836,12 @@ Sorry Giacom. Please don't be mad :(
 
 	else if(istype(loc, /obj/machinery/atmospherics/components/unary/cryo_cell))
 		var/obj/machinery/atmospherics/components/unary/cryo_cell/C = loc
-		var/datum/gas_mixture/G = C.airs["a1"]
+		var/datum/gas_mixture/C_air_contents = C.airs[1]

```

Comment:
no what
what are you doing pls stop
trust me you don't want to do this there's a good chance it'll break cryo because of my stupid associative list